examples/e2e/app-pages-router/open-next.config.ts

```diff
@@ -1,27 +1,8 @@
-import type { OpenNextConfig } from "@opennextjs/aws/types/open-next.js";
+import { defineCloudflareConfig } from "@opennextjs/cloudflare";
 import kvCache from "@opennextjs/cloudflare/kv-cache";
```

Comment:
nit: across the files this is imported as `cache`, `kvCache`, or `incrementalCache`. Maybe you can make the name consistent in this PR?

Reply:
done